### PR TITLE
[Components] Zoho Meeting #7423

### PR DESCRIPTION
**Pull request by lcaresia**
@malexanderlim I can't test webinar actions because a OAuth scope is missing.

image

### <samp>🤖 Generated by Copilot at 069e83e</samp>

This pull request adds a new Zoho Meeting component to the Pipedream platform, which enables users to create and get meetings and webinars using the Zoho Meeting API. The pull request adds four actions, a package.json file, and an app file to the `components/zoho_meeting` folder. The pull request also updates the `pnpm-lock.yaml` file with new dependencies. The pull request deletes two unused files from the `components/zoho_meeting` folder.

### <samp>🤖 Generated by Copilot at 069e83e</samp>

> _To make Zoho Meeting more fun_
> _We added some actions with `run`_
> _They use the `app` file_
> _And the `axios` style_
> _To create and to list webinar and meeting_

### <samp>🤖 Generated by Copilot at 069e83e</samp>

*  Add four new actions to the Zoho Meeting component for creating and getting meetings and webinars using the Zoho Meeting API ([link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-f23c861b469cd91fd5c3ce4cca72e34774945ee120b3bd1755bf8d4dbb38e516R1-R60), [link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-f4c58921f8e5ec15a63ce72cc387a38f2e9b4d80bd8ce27b23fc4745a53feb79R1-R60), [link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-7b5d27f131df6969084ea31f04fe1b7d63b0fc7293224590192e7cbbf9ebcca9R1-R42), [link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-ed65da0bf9145efdbd5707bb91984f72512c559505d7dbe752b8b142e8164d9dR1-R26))
* Add a new app file to the Zoho Meeting component that defines the common methods and properties for interacting with the Zoho Meeting API ([link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-fcbe221e7980c569bba829c33de66b4173c8ab26defc381b5c41c2a38c933aa3R1-R83))
* Update the package.json file of the Zoho Meeting component to include the `@pipedream/platform` dependency, update the version to 0.0.2, and update the main entry point to `zoho_meeting.app.mjs` ([link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-34fb53ae82470c7896a71797b9b4348dbcf0c7bdaa46c6eb243caeb908064937L3-R18))
* Update the pnpm-lock.yaml file to include the dependency information for the Zoho Meeting component and the jest-config package ([link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbL5636-R5640), [link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-32824c984905bb02bc7ffcef96a77addd1f1602cff71a11fbbfdd7f53ee026bbR18328-R18365))
* Delete the unused files `components/zoho_meeting/app/zoho_meeting.app.ts` and `components/zoho_meeting/.gitignore` ([link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-355ea5e7ba6614bff8aafedc20cb33ffa036524697e8c3dcf38e46faf158f68a), [link](https://github.com/PipedreamHQ/pipedream/pull/7453/files?diff=unified&w=0#diff-9c6a197692191d1430d10cdb69f6e04a0a4e539e2112583d5b5376696f86e8e4))

### CODE REVIEW

**Comment by malexanderlim** [comment]
Added the scope `ZohoMeeting.webinar.CREATE`.

File: components/zoho_meeting/actions/create-meeting/create-meeting.mjs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+import app from "../../zoho_meeting.app.mjs";
+
+export default {
+  name: "Create Meeting",
+  version: "0.0.1",
+  key: "zoho_meeting-create-meeting",
+  description: "Creates a meeting. [See docs here](https://www.zoho.com/meeting/api-integration/meeting-api/create-a-meeting.html)",
```
Comment (vellames) [inline components/zoho_meeting/actions/create-meeting/create-meeting.mjs:7]: Could you please change to `See the documentation`?

File: components/zoho_meeting/actions/create-webinar/create-webinar.mjs [hunk 2]
```diff
@@ -0,0 +1,60 @@
+import app from "../../zoho_meeting.app.mjs";
+
+export default {
+  name: "Create Webinar",
+  version: "0.0.1",
+  key: "zoho_meeting-create-webinar",
+  description: "Creates a webinar. [See docs here](https://www.zoho.com/meeting/api-integration/webinar-api/create-a-webinar.html)",
```
Comment (vellames) [inline components/zoho_meeting/actions/create-webinar/create-webinar.mjs:7]: Could you please change to `See the documentation`?

File: components/zoho_meeting/actions/get-meetings/get-meetings.mjs [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import app from "../../zoho_meeting.app.mjs";
+
+export default {
+  name: "Get Meetings",
+  version: "0.0.1",
+  key: "zoho_meeting-get-meetings",
+  description: "Get a list of meetings. [See docs here](https://www.zoho.com/meeting/api-integration/meeting-api/list-of-meeting-api.html)",
```
Comment (vellames) [inline components/zoho_meeting/actions/get-meetings/get-meetings.mjs:7]: Could you please change to `See the documentation`?

File: components/zoho_meeting/actions/get-webinars/get-webinars.mjs [hunk 4]
```diff
@@ -0,0 +1,26 @@
+import app from "../../zoho_meeting.app.mjs";
+
+export default {
+  name: "Get Webinars",
+  version: "0.0.1",
+  key: "zoho_meeting-get-webinars",
+  description: "Get a list of webinars. [See docs here](https://www.zoho.com/meeting/api-integration/webinar-api/list-of-webinar-api.html)",
```
Comment (vellames) [inline components/zoho_meeting/actions/get-webinars/get-webinars.mjs:7]: Could you please change to `See the documentation`?

File: components/zoho_meeting/package.json [hunk 5]
```diff
@@ -1,16 +1,18 @@
 {
   "name": "@pipedream/zoho_meeting",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (vellames) [inline components/zoho_meeting/package.json:3]: 0.1.0

**Review by vellames — CHANGES_REQUESTED** [review]
Hi there. Just a few minor changes

File: components/zoho_meeting/actions/get-meetings/get-meetings.mjs [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+  type: "action",
+  props: {
+    app,
+    index: {
+      label: "Index",
+      description: "The index or position of first meeting to list",
+      type: "integer",
+      optional: true,
+      default: 0,
+    },
```
Comment (michelle0927) [inline components/zoho_meeting/actions/get-meetings/get-meetings.mjs:17]: We should handle pagination within the action. `index` can be removed as a prop, and `listtype` should be added as a prop with the following choices: (all | past | week | month | today | upcoming) and with "all" as the default. The action should paginate through the meetings and return a maximum of `count` meetings.

**Review by michelle0927 — DISMISSED** [review]

**Review by michelle0927 — CHANGES_REQUESTED** [review]
No pagination was added. https://github.com/PipedreamHQ/pipedream/pull/7453#discussion_r1281969232

**Review by michelle0927 — APPROVED** [review]

**Review by luancazarine — DISMISSED** [review]
Hi @lcaresia, thanks for your contribution! LGTM!

**Review by michelle0927 — APPROVED** [review]

**Pull request merged**